src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericTable.js

```diff
@@ -678,8 +678,8 @@ export const GenericTable = ({
       // To trigger isDirty state when an already saved table was reverted and avoid it in other cases, we need to
       // updateParameterValue setter and updateOnFirstEdition function that triggers the start of edition; on the other
       // hand, updateParameterValueWithReset setter rollbacks modified values without triggering isDirty
-      _getDataFromTwingraphDataset(parameter.id ? updateParameterValue : updateParameterValueWithReset);
-      if (parameter.id) updateOnFirstEdition();
+      _getDataFromTwingraphDataset(parameter.content ? updateParameterValue : updateParameterValueWithReset);
+      if (parameter.content) updateOnFirstEdition();
     },
     // eslint-disable-next-line
     [parameter.id, updateParameterValue, updateParameterValueWithReset, updateOnFirstEdition]
```

Comment:
In the dependencies array, should we replace `parameter.id` by `parameter.content`?
Also, maybe we should just disable the "exhaustive list" rule for eslint, instead of disabling eslint for the whole line ? :sweat_smile:

Reply:
Nice catch, thanks :sweat_smile: